crates/autopilot/src/shadow.rs

```diff
@@ -81,6 +76,7 @@ impl RunLoop {
             };
             observe::log_auction_delta(id, &previous, &auction);
             previous = Some(auction.clone());
+            self.liveness.auction();
```

Comment:
I think this is fine at the moment but I think eventually we should consider making the distinction between ready and healthy.
That way we can be more cautious on `/healthy` and only update its timestamp if `self.single_run` returned without an error (i.e. we are actually settling auctions) and `/healthy` can be updated whenever we created a new auction (i.e. a new `autopilot` instance is ready to start working).

Additionally we might run into issues with the current code if we don't have any open order because `self.next_auction()` would return `None`. Then we'd report the `autopilot` as unhealthy although everything works as it should.